bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java

```diff
@@ -27,6 +32,9 @@
 	private UserDao userDao;
 	@Reference
 	private UserAdmin userAdmin;
+	
+	@Reference
```

Comment:
If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove.